### PR TITLE
Automatically add the API_PREFIX if missing — Fixes #113

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: cliquet/tests/test_views_batch.py [hunk 1]
```diff
@@ -81,6 +71,16 @@ def test_batch_cannot_be_recursive(self):
         resp = self.app.post_json('/batch', body, status=400)
         self.assertIn('Recursive', resp.json['message'])
 
+    def test_responses_are_resolved_with_api_with_prefix(self):
```
Comment (ametaireau) [inline cliquet/tests/test_views_batch.py]: `are_resolved_with_api_prefix` ?

Comment (ametaireau) [inline cliquet/tests/test_views_batch.py]: or even, `test_batch_adds_api_prefix_if_not_present`

File: cliquet/views/batch.py [hunk 2]
```diff
@@ -116,7 +116,10 @@ def build_request(original, dict_obj):
     :param original: the original batch request.
     :param dict_obj: a dict object with the sub-request specifications.
     """
+    api_prefix = '/'.join(original.upath_info.split('/')[:2])
```
Comment (ametaireau) [inline cliquet/views/batch.py]: If we have path_info = "/v1/toto/tata", then the split returns `['', 'v1', 'toto', 'tata']`

and then we join it again . We should do a format with a `"/{prefix}".format(prefix=original.upath_info.split('/')[1])`

File: cliquet/views/batch.py [hunk 3]
```diff
@@ -116,7 +116,10 @@ def build_request(original, dict_obj):
...
     path = dict_obj['path']
+    if not path.startswith(api_prefix):
+        path = '%s%s' % (api_prefix, path)
```
Comment (leplatrem) [inline cliquet/views/batch.py]: path = api_prefix + path  ?

File: cliquet/views/batch.py (hunk 2 above)
Comment (leplatrem) [inline cliquet/views/batch.py]: Can we put this comment in the code ?

File: cliquet/views/batch.py (hunk 3 above)
Comment (ametaireau) [inline cliquet/views/batch.py]: +1

**Pull request merged**